xikolo-ios/ViewControllers/Courses/WebViewController.swift

```diff
@@ -50,11 +50,12 @@ extension WebViewController : UIWebViewDelegate {
         let userIsLoggedIn = UserProfileHelper.isLoggedIn()
         let headerIsPresent = request.allHTTPHeaderFields?.keys.contains(Routes.HTTP_AUTH_HEADER) ?? false
 
-        if let url = request.url?.absoluteString, userIsLoggedIn && !headerIsPresent {
+        if userIsLoggedIn && !headerIsPresent {
             DispatchQueue.global().async {
                 DispatchQueue.main.async {
-                    let newRequest = NetworkHelper.getRequestForURL(url)
```

Comment:
check if this method is obsolete now and can be removed

Reply:
No, it is still used in `WebViewController`.